package.json

```diff
@@ -19,7 +19,10 @@
   "license": "MIT",
   "dependencies": {
     "@reactivex/rxjs": "^5.0.0-beta.12",
-    "promise": "^7.1.1"
+    "he": "^1.1.0",
+    "marked": "^0.3.6",
+    "promise": "^7.1.1",
+    "react": "^15.3.2"
```

Comment:
Why is react now being linked in? Until now we've been including it as a global, which I'd prefer to do, for better browser caching.